sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -572,6 +593,11 @@ export interface GetModelOptions extends OperationOptions {
 export interface GetOperationOptions extends OperationOptions {
 }
 
+// @public (undocumented)
+export interface HasBoundingPolygon {
```

Comment:
should this be documented?

Reply:
Good catch. @HarshaNalluru could you add a documentation comment to this interface?

Reply:
done